src/main/java/us/kilroyrobotics/RobotContainer.java

```diff
@@ -90,149 +89,65 @@ public class RobotContainer {
-            Commands.runOnce(() -> elevator.setPosition(ElevatorConstants.kL2Height), elevator);
-    private Command elevatorSetL3 =
-            Commands.runOnce(() -> elevator.setPosition(ElevatorConstants.kL3Height), elevator);
-    private Command elevatorSetL4 =
-            Commands.runOnce(() -> elevator.setPosition(ElevatorConstants.kL4Height), elevator);
-    private Timer backupTimer = new Timer();
-    private Command elevatorSetL4AndBackup =
-            Commands.parallel(
-                    elevatorSetL4,
-                    Commands.runOnce(
-                            () -> {
-                                this.backupTimer.restart();
-
-                                CommandScheduler.getInstance()
-                                        .schedule(
-                                                drivetrain
-                                                        .applyRequest(
-                                                                () ->
-                                                                        forwardStraight
-                                                                                .withVelocityX(
-                                                                                        FeetPerSecond
-                                                                                                .of(
-                                                                                                        -1)))
-                                                        .until(
-                                                                () ->
-                                                                        this.backupTimer.hasElapsed(
-                                                                                (0.25))));
-                            }));
-    private Command elevatorSetCoralStation =
-            Commands.runOnce(
-                    () -> elevator.setPosition(ElevatorConstants.kCoralStationHeight), elevator);
-
-    /* Wrist Commands */
-    private Command wristSetL1 =
-            Commands.runOnce(() -> wrist.setAngle(CoralMechanismConstants.kScoringL1), wrist);
-    private Command wristSetL2 =
-            Commands.runOnce(() -> wrist.setAngle(CoralMechanismConstants.kScoringL2), wrist)
-                    .andThen(Commands.runOnce(() -> leds.setMode(LEDMode.Rainbow), leds));
-    private Command wristSetL3 =
-            Commands.runOnce(() -> wrist.setAngle(CoralMechanismConstants.kScoringL3), wrist);
-    private Command wristSetL4 =
-            Commands.runOnce(() -> wrist.setAngle(CoralMechanismConstants.kScoringL4), wrist);
-    public Command wristSetCoralStation =
-            Commands.runOnce(() -> wrist.setAngle(CoralMechanismConstants.kIntakingAngle), wrist)
-                    .andThen(Commands.runOnce(() -> leds.setMode(LEDMode.Rainbow), leds));
+                        elevator.setPosition(elevator.getPosition());
```

Comment:
For wristStop, you have wrist.stop. But you do not have that for elevatorStop, why?

Reply:
elevator.stop() will kill power to it, so it'll succumb to gravity

wrist.stop() is fine because it seems to stay in place and the motor isn't working hard to keep it there (presets aren't calling wrist.stop() so even though they dip down a bit we're good)

Reply:
Ok let's remove wrist.stop() then from wristStop